all/099-k3s.yml

```diff
@@ -31,14 +31,31 @@ k3s_node_ip: '{{ ansible_facts[k3s_interface]["ipv4"]["address"] }}'
 # Disable the taint manually by setting: k3s_master_taint = false
 k3s_master_taint: "{{ true if groups['k3s_node'] | default([]) | length >= 1 else false }}"
 
+# monitoring
+k3s_monitoring: true
+k3s_monitoring_scheduler_port: "10259"
+k3s_monitoring_controller_manage_port: "10257"
+
 # these arguments are recommended for servers as well as agents:
 extra_args: >-
   --flannel-iface={{ k3s_interface }}
   --node-ip={{ k3s_node_ip }}
 
+# monitoring arguments instruct k3s servers to expose control plane metrics:
+monitoring_args: >-
+  --etcd-expose-metrics
+  --kube-proxy-arg "metrics-bind-address=0.0.0.0"
```

Comment:
I would prefer to make this address configurable. We can use 0.0.0.0 by default, but an integrator should be able to change this without touching the monitoring_args. I hink a single parameter (monitoring_bind_address?) is enough.

Reply:
I agree, fix addressed within e5bd5749da9f92af5aa5e7653fe2232868425069 and tested using testbed